src/auro-popover.js

```diff
@@ -145,18 +135,18 @@ class AuroPopover extends LitElement {
   toggleShow() {
     this.popper.show();
     this.isPopoverVisible = true;
-    this.setAttribute('data-show', '');
+    this.setAttribute('data-show', true);
   }
 
   // function that renders the HTML and CSS into  the scope of the component
   render() {
     return html`
       <div id="popover" class="popover util_insetLg">
         <div id="arrow" class="arrow" data-popper-arrow></div>
-        <slot role="tooltip"></slot>
+        <slot role="tooltip" aria-live="assertive"></slot>
```

Comment:
When testing with a screen reader, this is not reading back the content when the popover appears. 

I'd suggest putting `aria-live="polite"` on line 144.

Reply:
Good idea, making this change.